staff/emma-cowdery/horroy/scripts/horroy.js

```diff
@@ -0,0 +1,192 @@
+function Horroy() {
```

Comment:
You should check all the validations, in some methods you didn't pay attention to them 🤓